.storybook/pages/CoursePlannerStep1/CoursePlannerStep1.tsx

```diff
@@ -23,10 +23,10 @@ import CompassCenter from '../../static/compass-center.svg';
 
 export const CoursePlannerStep1 = () => (
   <>
-    <Panel className="u-margin-bottom-xl" variant="squared">
+    <Panel className="mb-8" variant="squared">
       <LayoutContainer>
         <PageHeader
-          className="u-margin-bottom-none"
+          className="mb-0"
```

Comment:
```suggestion
          className="!mb-0"
```
Same as the [course planner edit page comment](https://github.com/chanzuckerberg/edu-design-system/pull/1322/files#r983726151).